Function/BookFunc/Helpers/Constants.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TeamsMeetingBookFunc.Helpers
+{
+    internal class ConfigConstants
+    {
+        internal const string TenantIdCfg = "TenantID";
+        internal const string ClientIdCfg = "ClientID";
+        internal const string UserPasswordCfg = "UserPassword";
+        internal const string UserEmailCfg = "UserEmail";
+        internal const string DefaultMeetingNameCfg = "DefaultMeetingName";
```

Comment:
Add a comment to remember that if we change the values of those constants, we MUST update the ARM template as well

Reply:
```suggestion
        // When changing values here, please update the ARM template
        internal const string TenantIdCfg = "TenantID";
        internal const string ClientIdCfg = "ClientID";
        internal const string UserPasswordCfg = "UserPassword";
        internal const string UserEmailCfg = "UserEmail";
        internal const string DefaultMeetingNameCfg = "DefaultMeetingName";
```